test/unit/rules/exports-valid.test.js

```diff
@@ -0,0 +1,49 @@
+const ruleModule = require('../../../src/rules/exports-valid');
+
+const {lint, ruleType} = ruleModule;
+
+describe('exports-valid Unit Tests', () => {
+  describe('a rule type value should be exported', () => {
+    test('it should equal "standard"', () => {
+      expect(ruleType).toStrictEqual('standard');
```

Comment:
I don't know what this is, really.

Reply:
We've been doing this as a sanity check that the rule type is configure correctly. The rule type allows each rule to accept a standard `error`/`warning`/`off` config, a config object, a list of values, or an optional config object.